### PR TITLE
added ws example

### PR DESCRIPTION
**Pull request by itsdeka**
### Description:
Added an example to show how it is possible to spawn multiple bfx ws instances to comply with the open subscriptions number constraint (max. 25)

### Breaking changes:
- [ ]

### New features:
- [ ]

### Fixes:
- [ ]

### PR status:
- [X] Version bumped
- [X] Change-log updated

### CODE REVIEW

File: bfxapi/examples/ws/multiple_instances.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    print(f"Instance {instance.id} is dead, removing it from instances list")
+    instances.pop(instance.id)
+
+
+def run():
+    loop = asyncio.get_event_loop()
+    task = Routine(loop, ws, interval=5)
+    loop.run_until_complete(task)
+
+run()
```
Comment (robertkowalski) [inline bfxapi/examples/ws/multiple_instances.py:181]: missing newline on end of file

(you can set your editor to automatically add them)

Comment (itsdeka) [reply bfxapi/examples/ws/multiple_instances.py:181]: ty

Comment (itsdeka) [reply bfxapi/examples/ws/multiple_instances.py]: done

**Comment by robertkowalski** [comment]
works like a charm, many thanks!

**Pull request merged**